rfc/0020-policy-group.md

```diff
@@ -275,6 +276,11 @@ Also, we could consider creating a [custom package](https://rhai.rs/book/rust/pa
 The policy server will support [raw policy](https://docs.kubewarden.io/tutorials/writing-policies/wasi/raw-policies) groups out of the box, since the group evaluation logic is implemented in the `EvaluationEnvironment`.
 It will be possible to define a policy group with raw policies only, and evaluate the expression by calling the `validate_raw/<group name>` endpoint.
 
+## Development
+
+The `kwctl` tool will be extended to support the policy groups, to allow users to develop and test policy groups locally.
```

Comment:
@kubewarden/kubewarden-developers 
it's worth to mention how the `kwctl` cli will run policy groups. 
Do you have any idea on how this could look like?